src/main/scala/centaur/test/workflow/WorkflowData.scala

```diff
@@ -53,7 +53,7 @@ object WorkflowData {
   implicit class EnhancedPath(val path: Path) extends AnyVal {
     /** Read an entire file into a string, closing the underlying stream. */
     def slurp: String = {
-      val source = io.Source.fromFile(path.toFile, "UTF-8")
+      val source = scala.io.Source.fromFile(path.toFile, "UTF-8")
```

Comment:
no, I can import it.  Wasn't related to my stuff just broke when I imported circe

PS What does ToL mean?